lilypad/_opentelemetry/_opentelemetry_outlines/__init__.py

```diff
@@ -0,0 +1,180 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Modifications copyright (C) 2024 Mirascope
```

Comment:
Do we need to cover apache license if all of this is our original code? I have the license only on the OpenAI instrumentation bc thats the only one we extend.
@willbakst

Reply:
If it's fully original code then it's covered under our primary MIT license and we don't need this (since it's original and not modifications to existing code under a different license)